### PR TITLE
Mobile menu pull request

### PR DESCRIPTION
**Pull request by christianonoh**
## 📱 Implementation of Mobile-Menu Page Using DOM

**Description:** This pull request updates the previous merged pages with a mobile menu page with DOM manipulations.
The page includes the following:
-  a script.js file.
- modifications in the index.html and CSS files to style the page.
- update the assets/images folder with background images.
- some interactive transitions.

Thanks

### CODE REVIEW

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @christianonoh , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.\



## Invalid Code Review Request

You have submitted a project, that has linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by rloterh — COMMENTED** [review]
Hi Burger,
 
 While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
 
 ## Invalid Code Review Request
You have submitted a project, whereby **ESLint** linter checks are not showing on your Pull Request. Kindly ensure that **all** required linters are properly configured with regard to GitHub actions, and **all** checks are passing. Kindly request another code review when this is done.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by christianonoh** [comment]
@rloterh @alexander16108 Please how do I make my ESLint run? I've tried everything including adding it to the main branch but it's still not working

File: index.html [hunk 1]
```diff
@@ -15,16 +15,17 @@
...
-        <ul id="menu-items" class="flex-list">
-          <li class="navbar-links display-flex-column"><a href="#">Hello</a></li>
-          <li class="navbar-links display-flex-column"><a href="#">Portfolio</a></li>
-          <li class="navbar-links display-flex-column"><a href="#">About</a></li>
-          <li class="navbar-links display-flex-column"><a href="#">Contact</a></li>
+        <div class="open-menu display-flex-column"><i class="fa fa-bars fa-xl" aria-hidden="true"></i></div>
+        <ul class="menu-items flex-list">
+          <li class="navbar-links display-flex-column"><a href="#home">Hello</a></li>
+          <li class="navbar-links display-flex-column"><a href="#projects">Portfolio</a></li>
+          <li class="navbar-links display-flex-column"><a href="#about">About</a></li>
```
Comment (Davidosky007) [inline index.html:22]: * Kindly use the right color for the mobile menu list as given in the Figma design 
## Figma design image
## Yours 
image

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @christianonoh 👋 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

* No linter errors 👍 
* Good looking readme file 👍 
* Proper use of Github flow 👍 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Burger, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**